bedrock/firefox/tests/test_views.py

```diff
@@ -992,13 +992,6 @@ def test_accounts_page_2019(self, render_mock):
         views.firefox_accounts(req)
         render_mock.assert_called_once_with(req, 'firefox/accounts-2019.html', ANY)
 
-    @patch('bedrock.firefox.views.l10n_utils.render')
-    @patch.object(views, 'lang_file_is_active', lambda *x: False)
-    def test_accounts_page_legacy(self, render_mock):
-        req = RequestFactory().get('/firefox/accounts/')
-        req.locale = 'fr'
-        views.firefox_accounts(req)
-        render_mock.assert_called_once_with(req, 'firefox/accounts.html', ANY)
 
```

Comment:
Oh, it looks like there's one last flake 8 warning here:

```
./bedrock/firefox/tests/test_views.py:997:1: E303 too many blank lines (3)
1     E303 too many blank lines (3)
```

I believe it wants each test to be separated by 2 blank lines, but you have 3 here. Deleting this line should fix it, then we're good to merge.